BootloaderCommonPkg/Include/Guid/SmmInformationGuid.h

```diff
@@ -81,7 +81,8 @@ typedef struct {
 typedef struct {
   UINT8                       Revision;
   UINT8                       Flags;
-  UINT8                       Reserved[2];
+  UINT8                       Reserved[1];
+  UINT8                       BlSwSmiHandlerInput;

```

Comment:
Prefer passing BlSwSmiHandlerInput to UEFI payload from the communication area.
This HOB is shared by all bootloaders. e.g. coreboot doesn't need pass this info to UEFI payload.

Reply:
I think this info should be passed from payload to SBL instead.   We don't need to pass from SBL to payload.

Reply:
modified the code to reflect your comment